doc/source/examples/multiphysics/dam-break-VOF/dam-break-VOF.rst

```diff
@@ -0,0 +1,255 @@
+==========================
+Dam break VOF
+==========================
+
+This example simulates the dam break experiments of `Martin and Moyce (1952)`_. 
+
+.. _Martin and Moyce (1952): https://royalsocietypublishing.org/doi/abs/10.1098/rsta.1952.0006
+
+A liquid is fixed behind a dam at the left most corner of
+a rectangular domain as shown in the figure below.
+At :math:`t = 0` seconds, the dam is removed, and 
+the liquid is released into the total simulation domain. 
+The corresponding parameter file is 
+``gls_VOF_dam-break_Martin_and_Moyce.prm``.
+
+The following schematic describes the simulation:
+
+.. image:: images/VOF_dam_break_configuration.png
+    :alt: Schematic
+    :align: center
+
+.. note:: 
+    All the four boundary conditions are slip, and an external 
+    gravity field of :math:`-1`` is applied in the y direction.
+
+--------------
+Parameter file
+--------------
+
+Time integration is handled by a 1st order backward differentiation scheme 
+`(bdf1)`, for a :math:`4.1` seconds simulation time with an initial 
+time step of :math:`0.01` seconds.
+
+.. note::   
+    This example uses an adaptive time-stepping method in which the 
+    time-steps are modified during the simulation to keep the max CFD 
+    below a given threshold.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+        set method                         = bdf1
+        set time end                       = 4.1
+        set time step                      = 0.01
+        set adapt                          = true
+        set max cfl                        = 0.5
+        set stop tolerance                 = 1e-5
+        set adaptative time step scaling   = 1.1
+        set number mesh adapt              = 0
+        set output name                    = dam-break_VOF
+        set output frequency               = 20
+        set output path                    = ./Output/
+        set subdivision                    = 1      
+    end
+
+Make sure to create a directory named `Output` in the same directory 
+you are calling the solver from.   
```

Comment:
```suggestion
.. warning::
Make sure to create a directory named `Output` in the same directory 
you are calling the solver from.  Otherwise, the solver will be unable to generate the results files and will hang.
```